src/experimental/Toast/Toast.features.stories.tsx

```diff
@@ -0,0 +1,93 @@
+import React, {Meta} from '@storybook/react'
+import BaseStyles from '../../BaseStyles'
+import ThemeProvider from '../../ThemeProvider'
+import {Box, Text} from '../..'
+import {ToastUI} from './ToastUI'
+
+const meta: Meta = {
+  title: 'Components/Toast/Features',
+  decorators: [
+    Story => {
+      return (
+        <ThemeProvider>
+          <BaseStyles>
+            <Box sx={{display: 'inline-flex', flexDirection: 'column', gap: 3}}>{Story()}</Box>
+          </BaseStyles>
+        </ThemeProvider>
```

Comment:
```suggestion
            <Box sx={{display: 'flex', flexDirection: 'column', gap: 3}}>{Story()}</Box>
```

Should we consider some kind of ToastContainer component that will handle layout?

Reply:
in the future yes, but not today! So far this quite simply just has the individual toast ui component. The container will be managed by the up-and-coming `ToastOutlet` component, 🚧